lib/php-version.js

```diff
@@ -7,6 +7,7 @@
 
 const {listCompare} = require('./version.js');
 const {omitv} = require('./text-formatters.js');
+const lodash = require('lodash');
```

Comment:
I'm not sure which library is pulling in `lodash`, but since the declared dependency is `lodash.uniq` that's the one you should `require`:
```js
const uniq = require('lodash.uniq');
```

Reply:
fixed